.github/workflows/nucliadb.yml

```diff
@@ -33,6 +33,7 @@ jobs:
       matrix:
         python-version: ["3.11"]
         maindb_driver: ["tikv", "pg"]
+        release_channel: ["EXPERIMENTAL", "STABLE"]
```

Comment:
To not have an explosion of jobs in CI, maybe we can limit the release channel tests to run always with maindb_driver=tikv?

Reply:
do you want to have these tested?

- tikv+EXPERIMENTAL
- tikv+STABLE
- pg+STABLE

or just:

- tikv+EXPERIMENTAL
- pg+STABLE

Reply:
I guess 

- tikv+EXPERIMENTAL
- tikv+STABLE
- pg+STABLE

is better so we don't have to wonder which is the culprit for failures

but it also shouldn't be difficult to separate tikv from index node issues.